src/main/java/lotto/view/InputView.java

```diff
@@ -15,13 +17,32 @@ public static int inputPurchaseAmount() {
     }
 
     public static String inputWinningNumbers() {
+        System.out.println();
         System.out.println("지난 주 당첨 번호를 입력해 주세요.");
```

Comment:
줄바꿈을 위해 `System.out.println()`이 여러 번 추가로 호출되고 있는데
메시지를 출력할 때 줄바꿈 문자를 전달하여 한 번만 호출해보면 어떨까요?
플랫폼에 독립적인 줄바꿈 문자를 사용해보시는 것도 좋을 것 같아요.
https://www.baeldung.com/java-string-newline
```suggestion
        System.out.println(System.lineSeparator() + "지난 주 당첨 번호를 입력해 주세요.");
```